components/mdx/Code.tsx

```diff
@@ -1,192 +1,173 @@
 // @ts-nocheck
 
 import React from "react";
-import Highlight, { defaultProps } from "prism-react-renderer";
-import { Prism } from "prism-react-renderer";
-import rangeParser from "parse-numeric-range";
-import classnames from "classnames";
+import Highlight, { Prism, defaultProps } from "prism-react-renderer";
+import classNames from "classnames";
 
 import FileIcon from "../icons/File";
 import CopyIcon from "../icons/Copy";
 import CopyButton from "../Copy";
-import classNames from "classnames";
 
 // Extend base classes
-(typeof global !== "undefined" ? global : window).Prism = Prism;
-require("prismjs/components/prism-docker");
+(typeof global === "undefined" ? window : global).Prism = Prism;
+import("prismjs/components/prism-docker");
```

Comment:
Does this not need to be awaited in some form?

Reply:
I thought I removed the parentheses 💀